src/Component/ClickPhoto.js

```diff
@@ -0,0 +1,43 @@
+import React , {useState} from 'react'
+import Score from './Score';
+
+function ClickPhoto() {
+
+  const [score,setscore] =useState(0);
```

Comment:
use camelCase for variable names

Reply:
there should be a space after `,`. Please configure prettier to auto-format the files on save